js/src/modal.js

```diff
@@ -443,11 +444,11 @@ class Modal {
     const isModalOverflowing =
       this._element.scrollHeight > document.documentElement.clientHeight
 
-    if (!this._isBodyOverflowing && isModalOverflowing) {
+    if ((!this._isBodyOverflowing && isModalOverflowing && !isRTL) || (this._isBodyOverflowing && !isModalOverflowing && isRTL)) {
       this._element.style.paddingLeft = `${this._scrollbarWidth}px`
     }
 
-    if (this._isBodyOverflowing && !isModalOverflowing) {
+    if ((this._isBodyOverflowing && !isModalOverflowing && !isRTL) || (!this._isBodyOverflowing && isModalOverflowing && isRTL)) {
```

Comment:
maybe to bit a more readable we can split those two `if` in something like that: 

```js
if (isRTL) {

} else {

}
```

Reply:
Not quite sure to understand this one: `this._isBodyOverflowing` and `isModalOverflowing` are opposite too. So a simpe `else` wouldn't match.

Reply:
I meant something like that:

```js
if (isRTL) {
  if (!this._isBodyOverflowing && isModalOverflowing) {
  
  }
} else {
  if (this._isBodyOverflowing && !isModalOverflowing) {
  
  }
}
```

But now I'm not sure if it will be better 🤔 

What do you think?

Reply:
This example only handles hafl of the cases; I find it very verbose when it's complete:

``
if (isRTL) {
  if (!this._isBodyOverflowing && isModalOverflowing) {
  
  } else if (this._isBodyOverflowing && !isModalOverflowing) {
  
  }
} else {
  if (!this._isBodyOverflowing && isModalOverflowing) {
  
  } else if (this._isBodyOverflowing && !isModalOverflowing) {
  
  }
}```

And it would require to duplicate `.paddingRight` and `.paddingLeft` setters. So even if it's more legible, I don't think it's better.

I'm not sure how to improve this (as I said, not my expertise) but being more verbose seems overcomplicated to me.

Reply:
forget about that 😉